.travis.yml

```diff
@@ -1,15 +1,16 @@
-language: node_js
-node_js:
-    - "0.10"
+language: php
+php:
+    - "5.6"
```

Comment:
> (5.7 ? I guess you mean 7 :))

Right :)

Thanks for the info, just curious if it was a typo or not, as it looked weird to me as well.